src/components/guest/DurationOfStaySubfields.js

```diff
@@ -0,0 +1,16 @@
+export class DurationOfStaySubfields {
+    static DIMENSIONLESS_VALUE = "dimensionlessDurationOfStay";
+    static UNIT = "timeUnit";
+    static #separator = " ";
+
+    static fromString(str) {
+        return {
+            [this.DIMENSIONLESS_VALUE]: parseInt(str),
+            [this.UNIT]: str.split(this.#separator)[1],
+        };
+    }
+
+    static toString(durationOfStay) {
+        return `${durationOfStay[this.DIMENSIONLESS_VALUE]}${this.#separator}${durationOfStay[this.UNIT]}`;
+    }
```

Comment:
It looks to me like normal string is easier to manage, than string that produced with momentum. 
As far as I see `momentum.humanize` function maps some values as human readable which produces more values (e.g. for month you can have months and month) and makes mapping from model to form and vice verce more complicated.
Also it introduces more complications in dropdown because you should select translation depending on provided dimensionless value.

@argasek , what do you think: should we stay with this implementation or try to leverage `momentum`?

Reply:
We can either utilize momentjs built-in functions or https://www.i18next.com/translation-function/plurals - whichever comes easier.

Reply:
Utilized plurals from `i18next`. Thanks for tip!